src/plugins/GenerateJob/GenerateJob.js

```diff
@@ -177,19 +177,16 @@ define([
                 const filename = GenerateJob.toSnakeCase(name);
                 const code = this.core.getAttribute(node, 'code');
 
-                return [filename, name, code];
+                return [filename, code];
             });
 
         operations.forEach(tuple => {
```

Comment:
Why don't we just change this so it only adds the given operation to the job files? This should then make it so we don't need to make any other code changes, I think.